components/CategoryCard.js

```diff
@@ -24,39 +26,27 @@ export default ({ id, name }) => {
     }
     :first-child {
```

Comment:
Since it is here. It is very wrong way to handle the images and divs. 
I would create array of objects with meta data which contains url, and whatever else needed. Then I would iterate through this array of objects to display divs with meta data.

What is you use actual images instead of background-image?